dev-tools/scripts/releaseWizard.yaml

```diff
@@ -955,37 +955,36 @@ groups:
     id: stage_maven
     title: Stage the maven artifacts for publishing
     vars:
+      git_sha: '{{ build_rc.git_rev | default("<git_sha>", True) | truncate(7,true,"") }}'
       dist_folder: solr-{{ release_version }}-RC{{ rc_number }}-rev-{{ build_rc.git_rev | default("<git_rev>", True) }}
     commands: !Commands
       root_folder: '{{ git_checkout_folder }}'
       confirm_each_command: true
       commands_text: In the source checkout do the following (note that this step will prompt you for your Apache LDAP credentials)
       commands:
         - !Command
-          cmd: ant clean stage-maven-artifacts  -Dmaven.dist.dir={{ [dist_file_path, dist_folder, 'solr', 'maven'] | path_join }}  -Dm2.repository.id=apache.releases.https  -Dm2.repository.url={{ m2_repository_url }}
+          cmd: bash dev-tools/scripts/upload-maven.sh -u {{ gpg.apache_id }}  -v {{ release_version }} -c {{ git_sha }} -d "{{ [dist_file_path, dist_folder, 'solr', 'maven'] | path_join }}"
+          tee: true
           logfile: publish_solr_maven.log
     post_description: The artifacts are not published yet, please proceed with the next step to actually publish!
   - !Todo
     id: publish_maven
     depends: stage_maven
     title: Publish the staged maven artifacts
+    vars:
+      git_sha: '{{ build_rc.git_rev | default("<git_sha>", True) | truncate(7,true,"") }}'
```

Comment:
Will do. For the 9.0 release, you will need to manually run this command (because the script won't exist in the repo, and because the build_rc task will not have the git_rev_short variable).

Reply:
> Will do. For the 9.0 release, you will need to manually run this command (because the script won't exist in the repo, and because the build_rc task will not have the git_rev_short variable).

Sure, no prob.